src/Symfony/Component/Cache/Traits/PhpArrayTrait.php

```diff
@@ -126,6 +139,6 @@ public function clear()
      */
     private function initialize()
     {
-        $this->values = file_exists($this->file) ? (include $this->file ?: array()) : array();
+        list($this->keys, $this->values) = file_exists($this->file) ? (include $this->file ?: array(array(), array())) : array(array(), array());
```

Comment:
what happens in case we have a file which was dumped with the old implementation ? woudln't this break here ?

Reply:
note that I'm totally fine with the fact that an old file is not usable anymore. But I want this case to be handled as if the file was not there at all, not by having a broken cache.

Reply:
now taken into account